percy/connection.py

```diff
@@ -15,6 +15,13 @@ def get(self, path, options={}):
             'User-Agent': self.user_agent,
         }
         response = requests.get(path, headers=headers)
+        try:
+            response.raise_for_status()
+        except Exception as e:
+            print('Received a {} error requesting: {}'
```

Comment:
Style: this could be one line.

Also, we for consistency with other error messages (and to output to stderr rather than stdout), use:

```python
from percy import utils

utils.print_error('[percy] Error: ...')
```